packages/database/src/modelClasses/FeedItem.js

```diff
@@ -27,4 +27,60 @@ export class FeedItemModel extends DatabaseModel {
   get DatabaseTypeClass() {
     return FeedItemType;
   }
+
+  /**
+   *
+   * @param {AccessPolicy} accessPolicy
+   * @param {string} projectId
+   * @param {object} conditions
+   * @param {object} options
+   * @param {string[]} [options.sort]
+   * @param {number} [options.pageLimit]
+   * @param {number} [options.page]
+   * @returns
+   */
+  async findFeedItemsByAccessPolicy(accessPolicy, projectId = '', conditions = {}, options = {}) {
```

Comment:
```suggestion
  async findByAccessPolicy(accessPolicy, projectId = '', conditions = {}, options = {}) {
```
Same thinking with the method rename here